fix: Include new table view settings (#20609)
newrelic/docs-website: src/content/docs/logs/ui-data/use-logs-ui.mdx

<img
  title="Logs Export UI with details"
  alt="Screenshot of Export UI with details"
  src="/images/logs_crop_export.webp"
/>

## Customizable table views [#customizable-table-views]

> Take control of how you interact with your log data using the table settings.

Reviewer: ```suggestion
Customize how you interact with your log data using the <DNT>Table settings</DNT> option.
```

<img
  title="Logs Table Settings menu"
  alt="Screenshot of Table Settings menu"
  src="/images/logs_crop_table_settings_menu.webp"
/>

### Set the Log View: 

> Choose how the rows in your log table are displayed:

Reviewer: ```suggestion
Select how log entries are displayed in the table:
```

- **Condensed:** Display each log entry in a single line, allowing for rapid scanning of a large volume of logs.

> - **Expanded:** View the full content of each log entry by default, providing immediate access to all details without needing to click.

Reviewer: ```suggestion
- **Expanded:** Display the full content of each log entry by default for immediate access to all details.
```

### Set the Log Detail Exploration: 

> Select your preferred method for examining individual log entries:

Reviewer: ```suggestion
Select how to view the details of individual log entries:
```

> - **Side Panel:** Clicking a log row will open a convenient slide-out panel from the right, displaying the complete log details including logs in context.

Reviewer: ```suggestion
- **Side Panel:** Select a log entry to open a side panel on the right. The panel displays complete log details, including logs in context.
```

<img
  title="Logs side panel detail view"
  alt="Screenshot of log side panel detail view"
  src="/images/logs_crop_side_panel_detail_view_expanded.webp"
/>

> - **Table Row:** Clicking a log row will expand it directly within the table, revealing the full log content. If the log message contains a JSON object, it will be automatically formatted and prettified for easier analysis. Additionally, a "Show log detail" button will appear on hover, providing an alternative way to open the side panel.

Reviewer: ```suggestion
- **Table Row:** Select a log entry to expand it within the table and view the full log content. If a log message contains a JSON object, it is automatically formatted for easier analysis. Alternatively, hover over the log row and click <DNT>**Show log detail**</DNT> to open the side panel.
```
